samples/crypto/psa_tls/src/psa_tls_functions_client.c

```diff
@@ -41,27 +41,31 @@ static int setup_tls_client_socket(void)
 	sock = socket(AF_INET, SOCK_STREAM, IPPROTO_TLS_1_2);
 	if (sock < 0) {
 		LOG_ERR("Failed to create a socket. Err: %d", errno);
+		(void)close(sock);
```

Comment:
I don't think close(sock) here is valid (the other places seem ok).

close may also overwrite errno so that may add a surprise for someone

Reply:
Good point, removed it